pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <groupId>com.edmunds</groupId>
     <artifactId>databricks-rest-client</artifactId>
-    <version>2.4.3-SNAPSHOT</version>
+    <version>2.5.2-SNAPSHOT</version>
```

Comment:
this will actually need to go up to 2.6.1

Reply:
Done